.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-extend-ignore = E203, E266, E501
+ignore = E203, E266, E501, W503
```

Comment:
Per my understanding, it is already getting enabled by the `select` configuration below because we replace the default ignore list (that contains W504) with our own.

Reply:
Yeah that's probably true 😅 damn I'm still getting confused. Anyways, thanks a bunch for the contribution!